src/graphqlify.ts

```diff
@@ -40,3 +40,7 @@ export const graphqlify = {
   mutation: createOperate('mutation'),
   subscription: createOperate('subscription'),
 }
+
+export const alias = (name: string, query: string) => {
+  return `${name}: ${query}`
+}
```

Comment:
Could you execute `prettier --write src/*.ts`? I will add pre-commit hook but for the time being manually execute it.